part-ii-domain-logic/sagas.md

```diff
@@ -142,16 +142,25 @@ Axon supports life cycle management through the `AnnotatedSagaManager`, which is
 
 When using the Configuration API, Axon will use sensible defaults for most components. However, it is highly recommended to define a `SagaStore` implementation to use. The `SagaStore` is the mechanism that 'physically' stores the Saga instances somewhere. The `AnnotatedSagaRepository` \(the default\) uses the `SagaStore` to store and retrieve Saga instances as they are required.
 
-```java
+{% codetabs name="Axon Configuration API", type="java" -%}
 Configurer configurer = DefaultConfigurer.defaultConfiguration();
 configurer.registerModule(
-        SagaConfiguration.subscribingSagaManager(MySagaType.class)
+        SagaConfiguration.subscribingSagaManager(MySaga.class)
                          // Axon defaults to an in-memory SagaStore, defining another is recommended
                          .configureSagaStore(c -> new JpaSagaStore(...)));
 
 // alternatively, it is possible to register a single SagaStore for all Saga types:
 configurer.registerComponent(SagaStore.class, c -> new JpaSagaStore(...));
-```
+{%- language name="Spring Configuration", type="java" -%}
+@Saga(sagaStore = "mySagaStore")
+public class MySaga {...}
+...
+// somewhere in configuration
+@Bean
+public SagaStore mySagaStore() {
+    return new JpaSagaStore(...);
```

Comment:
I think using JPA as an example here may be confusing, as it is the default SagaStore implementation when JPA is detected. It might be worth mentioning that.

Reply:
You're right! I'll go for things that are not default...